plugins/hls-hlint-plugin/src/Ide/Plugin/Hlint.hs

```diff
@@ -512,8 +513,8 @@ ignoreHint _recorder ideState nfp verTxtDocId ignoreHintTitle = do
                   (Just (M.singleton (verTxtDocId ^. LSP.uri) textEdits))
                   Nothing
                   Nothing
-        pure $ Right workspaceEdit
-    Nothing -> pure $ Left  "Unable to get fileContents"
+        pure workspaceEdit
+    Nothing -> throwE $ PluginInternalError "Unable to get fileContents"
```

Comment:
Ah, I think all `throwE`s should be `throwError`s. Generally it's pretty good to use the `MonadError` stuff, it's very standard and people will recognize it.